packages/remix-eslint-config/internal.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * This config is intended for internal Remix projects. It should not be
+ * documented nor considered public API in regards to semver considerations.
+ */
+
+/**
+ * @see https://github.com/eslint/eslint/issues/3458
+ * @see https://www.npmjs.com/package/@rushstack/eslint-patch
+ */
+require("@rushstack/eslint-patch/modern-module-resolution");
+
+const OFF = 0;
+const WARN = 1;
+const ERROR = 2;
+
+module.exports = {
+  root: true,
+  extends: [
+    require.resolve("./index.js"),
+    require.resolve("./jest-testing-library.js"),
```

Comment:
I'm keeping the markdown rules/plugin out of the shared config for now because we'll be using this in outside Remix repos that won't use them. Might move them in later.